src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__college_assessment_dashboard.sql

```diff
@@ -150,13 +200,22 @@ select
 
     o.scale_score,
     o.rn_highest,
+
+    c.courses_course_name as subject_course,
+    c.teacher_lastfirst as subject_teacher,
+    c.sections_external_expression as subject_external_expression,
 from roster as e
 left join
-    act_sat_official as o
+    college_assessments_official as o
     on e.contact_id = o.contact
     and e.expected_test_type = o.test_type
     and e.expected_scope = o.scope
     and e.expected_subject_area = o.subject_area
+left join

```

Comment:
@GabyRangelB I think the linter error is referring to here: the `o` columns should be on the left and the `c` ones on the right

Reply:
you can reference the error docs here: https://docs.sqlfluff.com/en/stable/rules.html#rule-ST09